specs/concepts/verifyUserCanOperateStageForPipelineFromStageDetailsPage.cpt

```diff
@@ -0,0 +1,19 @@
+// --GO-LICENSE-START--
```

Comment:
Remove this license as its old, spec and cpt files dont need a license anyways

Reply:
For clarification: license is only needed on ruby code, not the gauge specs.